src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -1344,6 +1344,32 @@ smart_objects::SmartObjectList MessageHelper::CreateAddCommandRequestToHMI(
   return requests;
 }
 
+smart_objects::SmartObjectSPtr
+MessageHelper::CreateUnsubscribeVehicleDataMessageForHMI(
+    const VehicleInfoSubscriptions& vehicle_data,
+    const application_manager::ApplicationSharedPtr& app) {
```

Comment:
@IGapchuk We can use `ApplicationConstSharedPtr` instead of const reference

Reply:
@ZhdanovP Fixed in c85e66cb7cabc056ddd602e4803e96323969b5b0